src/applications/vaos/covid-19-vaccine/components/VAFacilityPage/FacilitiesRadioWidget.jsx

```diff
@@ -78,7 +78,11 @@ export default function FacilitiesRadioWidget({
                 {name}
               </span>
               <span className="vads-u-display--block vads-u-font-size--sm">
-                {address?.city}, <State state={address?.state} />
+                {address?.line}
```

Comment:
I wonder if using `FacilityAddress` would work here, since we're now showing the full address?

Also, `address.line` is an array, so this might look odd if a facility has a multi-line address.

Reply:
Updated to use `FacilityAddress`.